docs/security-and-privacy-design/netdata-cloud-security.md

```diff
@@ -41,10 +41,7 @@ Netdata Cloud does not store user credentials.
 
 ## Security Features and Response
 
-Netdata Cloud offers a variety of security features, including infrastructure-level dashboards, centralized alerts
-notifications, auditing logs, and role-based access to different segments of the infrastructure. The cloud service
-employs several protection mechanisms against DDoS attacks, such as rate-limiting and automated blacklisting. It also
-uses static code analyzers to prevent other types of attacks.
+Netdata Cloud offers a variety of security features, including infrastructure-level dashboards, centralized alerts notifications, auditing logs, and role-based access to different segments of the infrastructure. It employs several protection mechanisms against DDoS attacks, such as rate-limiting and automated blacklisting. It also uses static code analyzers to prevent other types of attacks.
```

Comment:
```suggestion
Netdata Cloud offers a variety of security features, including infrastructure-level dashboards, centralized alert notifications, auditing logs, and role-based access to different segments of the infrastructure. It employs several protection mechanisms against DDoS attacks, such as rate-limiting and automated blacklisting. It also uses static code analyzers to prevent other types of attacks.
```